src/biogeochem/CNPhenologyMod_cut_newFlux_newCutParam_N.F90

```diff
@@ -0,0 +1,3060 @@
+module CNPhenologyMod
```

Comment:
It seems that most of this is the same as CNPhenologyMod. Is there a reason you added an entirely new F90 routine instead of adding/modifying code in the CNPhenologyMod.F90?